checkpoint/cp-resource-management/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: v1.1.0
```

Comment:
should we start promoting appVersion here too?

Reply:
Not sure it's important for GA, and originally we wanted to promote it when we promote agent version. Till we have some solution on automatic upgrade.
But I can if you prefer